changelogs/unreleased/gh-8576-set-compat-options-to-new.md

```diff
@@ -0,0 +1,11 @@
+## feature/compat
```

Comment:
>     compat: switch existing options to new for tnt-3.0
>
>    Due to the approaching tarantool 3.0 release, the following compat
>    options' defaults were updated to new behavior:

Nit: I would not mention releasing 3.0 as a reason for a switching to a new behavior. A new release could happen or not, it could be 3.0 or 3.1 or even 4.0 this is not related to our intention to change behavior for these options.

My suggestion:

>   compat: switch existing options to a new behaviour
>   These compat options were switched to a new behavior:

Feel free to ignore.

Reply:
I think it is good to state the release, as compat options switch is bound with it. I will leave it as is.